content/plugin/framework/index.mdx

```diff
@@ -7,10 +7,9 @@ description: |-
 
 # Terraform Plugin Framework
 
-The plugin framework is a new way to develop Terraform plugins, providing improvements and new features from [Teraform Plugin SDKv2](/plugin/sdkv2/sdkv2-intro).
+The plugin framework is a new way to develop Terraform Plugins on [Protocol Version 6](../how-terraform-works#terraform-plugin-protocol). It offers improvements and new features from [Teraform Plugin SDKv2](/plugin/sdkv2/sdkv2-intro).
```

Comment:
@laurapacilio If I could make a recommendation, it would be to stick to urls that reference the full path

Ex. replace

```suggestion
The plugin framework is a new way to develop Terraform Plugins on [Protocol Version 6](/plugin/how-terraform-works#terraform-plugin-protocol). It offers improvements and new features from [Teraform Plugin SDKv2](/plugin/sdkv2/sdkv2-intro).
```

Reply:
No problem, will update any relative links and verify.